packages-content-model/roosterjs-content-model-editor/lib/publicTypes/ContentModelEditorCore.ts

```diff
@@ -1,5 +1,5 @@
 import type { ContentModelPluginState } from './pluginState/ContentModelPluginState';
-import type { CoreApiMap, EditorCore } from 'roosterjs-editor-types';
+import type { Border, CoreApiMap, EditorCore } from 'roosterjs-editor-types';
```

Comment:
For the new types that is only used by Content Model, declare it in `roosterjs-content-model-types` rather than `roosterjs-editor-types`.

Content Model will be moved out and totally get rid of old editor code.

Reply:
That would mean I need to import a content model type for MainPaneBase, which I currently do by mistake. If something does not depend on content model it should not import things from it.